### PR TITLE
fix : fixed table cell overflow in uploadExcel Model

### PR DESCRIPTION
**Pull request by musharaf-nocodb**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

**BEFORE**

Screenshot 2023-12-13 at 12 36 03 PM

**NOW**

Screenshot 2023-12-13 at 10 44 25 AM

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by dstala** [comment]
@musharaf-nocodb do we need this PR as you will be working on the new modal for import?

File: packages/nc-gui/components/template/Editor.vue [hunk 1]
```diff
@@ -1025,7 +1028,7 @@ watch(modelRef, async () => {
   }
 
   :deep(.template-form-row) > td {
-    @apply p-0 mb-0;
+    @apply !p-1 mb-0 truncate max-w-50;
```
Comment (wingkwong) [inline packages/nc-gui/components/template/Editor.vue:1031]: why do we need `!` to override here?

Comment (musharaf-nocodb) [reply packages/nc-gui/components/template/Editor.vue]: **fixed in below Commit**


[fix : removed padding important](https://github.com/nocodb/nocodb/pull/7221/commits/3b875d5bfc53b1c6bbe6b39f94fe592de808e5b3)
 
[3b875d5](https://github.com/nocodb/nocodb/pull/7221/commits/3b875d5bfc53b1c6bbe6b39f94fe592de808e5b3)

**Comment by musharaf-nocodb** [comment]
> @musharaf-nocodb do we need this PR as you will be working on the new modal for import?

its just UI fix.

**Pull request merged**